target/linux/ramips/dts/mt7620a_tplink_re200-v1.dts

```diff
@@ -0,0 +1,158 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,re200-v1", "ralink,mt7620a-soc";
+	model = "TP-Link RE200 v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		
+		led_power: power {
+		    label = "re200-v1:green:power";
+		    gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+		};
+		
+		lan {
+			label = "re200-v1:green:lan";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "re200-v1:green:wlan";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		};
+		
+		qss {
+		    label = "re200-v1:green:qss";
+		    gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_2g1 {
```

Comment:
What's this LED for?

Reply:
Green 2.4G LED, the other one is red. Original firmware shows the WiFi signal strength here.

Reply:
The use "wlan2_green" and "wlan2_red" or "wifi2_green" and "wifi2_red" or "wlan2g_green" etc.
Based on your choice of the name, also use "wifi" or "wlan" for the separate LED.

Reply:
done